### PR TITLE
Evaluation during training

### PR DESCRIPTION
**Pull request by 51N84D**
Evaluating validation masks during training using various metrics

### CODE REVIEW

**Comment by 51N84D** [comment]
Note: this was built on top of end-to-end training, so let's get that merged first

File: omnigan/eval_metrics.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+from PIL import Image
```
Comment (melisandeteng) [inline omnigan/eval_metrics.py:1]: Oh it did that for you too ! wonder why an empty file was pushed in the first place ...

File: shared/trainer/defaults.yaml [hunk 2]
```diff
@@ -1,4 +1,8 @@
 output_path: /network/tmp1/schmidtv/yb_runs/test_v1
+load_paths: 
+  p: none # none if using output_path
```
Comment (melisandeteng) [inline shared/trainer/defaults.yaml:3]: so we can't load from one place and save in another output_path ?

Comment (51N84D) [reply shared/trainer/defaults.yaml:3]: Currently not supported ;) we can make an issue if this becomes something we want to do

**Pull request merged**